docusaurus.config.js

```diff
@@ -7,11 +7,11 @@ const darkCodeTheme = require("prism-react-renderer/themes/dracula");
 /** @type {import('@docusaurus/types').Config} */
 const config = {
   title: "Substratus",
-  tagline: "Deploying and fine tuning LLMs made easy",
+  tagline: "Build, train, and serve LLMs and more!",
   favicon: "img/favicon.ico",
 
   // Set the production url of your site here
-  url: "https://substratus.ai",
+  url: "https://substratusai.github.io",
```

Comment:
Why not serve from substratus.ai?

Reply:
Worried about getting SEO dinged on the domain bc our links to substratus.git are broken